### PR TITLE
Migrate to Open gov

### PR DESCRIPTION
**Pull request by jeluard**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
```
Comment (hamidra) [inline .editorconfig:1]: Do we want to check in the editor config?!

File: src/types.ts [hunk 2]
```diff
@@ -1,9 +1,156 @@
...
-  index: number,
-  vote: boolean,
-  referendum: Referendum
-}
-
-export type Referendum = {
-  index: number,
-}
\ No newline at end of file
+import BigNumber from 'bignumber.js';
```
Comment (hamidra) [inline src/types.ts:1]: polkadot.js apis is using [BN.js](https://github.com/indutny/bn.js/). Any reason not to use that instead?!

File: .editorconfig (hunk 1 above)
Comment (jeluard) [reply .editorconfig:1]: https://editorconfig.org/ is a standard that is supported by most editors. You can see it as a way to have default config for editors, e.g. space/tabs, numbers of spaces, .. It's kind of a companion to code linters. 
I feel like it's common to check this in those days?

File: src/types.ts (hunk 2 above)
Comment (jeluard) [reply src/types.ts:1]: Good catch, I mixed my bignumbers :D

File: .editorconfig (hunk 1 above)
Comment (hamidra) [reply .editorconfig:1]: I see. Makes sense then. I'm also thinking to config ESlint later as we move forward.

**Review by hamidra — APPROVED** [review]

**Pull request merged**